lib/generate-suggestions.ts

```diff
@@ -3,58 +3,73 @@ import glob from "glob";
 import { parse } from "@babel/parser";
 import traverse from "@babel/traverse";
 
-import { fetchComponents } from "./http/fetchComponents";
+import {
+  FetchComponentResponseComponent,
+  fetchComponents,
+} from "./http/fetchComponents";
+
+interface Result extends FetchComponentResponseComponent {
```

Comment:
I came back and reorganized the return structure of `generate-suggestion` in a way that makes more sense for the vscode extension to consume. It's a lot easier to work with now!